src/Compilers/Core/Portable/Operations/OperationFactory.cs

```diff
@@ -81,6 +81,7 @@ public static IInvalidOperation CreateInvalidExpression(SemanticModel semanticMo
             return new InvalidOperation(children, semanticModel, syntax, type: null, constantValue: default(Optional<object>), isImplicit: isImplicit);
         }
 
+        public static Lazy<IOperation> EmptyOperation { get; } = new Lazy<IOperation>(() => null);
```

Comment:
>EmptyOperation [](start = 39, length = 14)

CSharpOperationFactory already has _lazyNullOperation field with exact same value. I think that field should be removed in favor of this one. Consider changing names to NullOperation and NullInitializer though.

Reply:
_lazyNothingOperation on VB side can be unified as well

---
In reply to: [149577775](https://github.com/dotnet/roslyn/pull/22972#discussion_r149577775) [](ancestors = 149577775)